docker-compose.release.yml

```diff
@@ -1,174 +1,60 @@
+# This file is used by .travis/make-github-release.sh to create a release bundle.
+# The script replaces all image names to include docker tag published to
+# dockerhub.io for the respective version.
 version: "3.4"
+
 services:
   proxy:
     image: fragaria/karmen-proxy
     restart: unless-stopped
-    network_mode: host
```

Comment:
`host` network was necessary for discovery of pill devices on local network. We dropped support for local installation of Karmen (formerly "Karmen Hub").
Moving the backend to backend network simplified configuration and made it much closer to what is used in production.

Reply:
We dropped Hub, but we still want people to run their own installation at their own RPi at home. Altrough these builds are now on hold, I don't think we should just kill this.

Reply:
It's not killed for sure. I just removed it from the default docker-compose.yml. Anyone can setup network configuration he wants.